pkg/rest/apisurface.go

```diff
@@ -23,8 +24,9 @@ import (
 type APISurface struct {
 	// Broker contains the business logic that provides the
 	// implementation for the different OSB API operations.
-	Broker  broker.Interface
-	Metrics *metrics.OSBMetricsCollector
+	Broker     broker.Interface
+	Metrics    *metrics.OSBMetricsCollector
+	EnableCORS bool
```

Comment:
Would be great to add some comment docs for this, above this line.

Reply:
I would take the doc comment in a follow-up. Are you LGTM otherwise?

Reply:
Sure 👍 But let's wait for @carolynvs as she had a comment.